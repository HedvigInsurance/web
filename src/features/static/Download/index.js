import React from "react"
import { Helmet } from "react-helmet";
import Lazyload from "react-lazyload"

import { SayHi } from "../common"
import Header from "../../../components/Header";
import Footer from "../../../components/Footer"
import "./download.css"

export default class Download extends React.Component {
  render() {
    return (
      <main className="Download">
        <Helmet>
          <title>Ladda ner appen | Hedvig</title>
        </Helmet>
        <Header headerRight={<SayHi/>}/>
        <article className="pure-g pure-centered Download__article">
          <div className="pure-u-1-1">
            <h1 className="Download__page-header">Ladda ner appen</h1>
            <h2 class="Download__section-header">Sök efter Hedvig i din Store för att komma&nbsp;igång.</h2>
          </div>
          <div className="pure-u-1-1">
            <section className="pure-g pure-centered">
              <a
                href="https://itunes.apple.com/se/app/hedvig/id1303668531?mt=8"
                target="_blank"
                rel="noopener noreferrer"
                className="Download__social-icon"
              >
                <Lazyload
                  height={46}
                  offset={200}
                >
                  <img
                    src="/assets/web/appstores/app-store-badge@2x.png"
                    alt="Ladda ner på App Store"
                    height={54}
                  />
                </Lazyload>
              </a>
              <a
                href="https://play.google.com/store/apps/details?id=com.hedvig.app"
                target="_blank"
                rel="noopener noreferrer"
                className="Download__social-icon"
              >
                <Lazyload
                  height={46}
                  offset={200}
                >
                  <img
                    src="/assets/web/appstores/google-play-badge@2x.png"
                    alt="Ladda ner på Google Play"
                    height={54}
                  />
                </Lazyload>
              </a>
            </section>
          </div>
        </article>
        <Footer />
      </main>
    )
  }
}
