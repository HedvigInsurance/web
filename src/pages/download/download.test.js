import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Download from './index';

describe('<Download />', () => {
  it('Should render correctly', () => {
    const component = shallow(<Download />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
