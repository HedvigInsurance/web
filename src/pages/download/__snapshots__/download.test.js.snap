// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Download /> Should render correctly 1`] = `
<main
  className="Site"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Ladda ner appen | Hedvig
    </title>
  </HelmetWrapper>
  <Header
    isInverted={false}
  />
  <article
    className="Site-content"
  >
    <div
      className="Container"
    >
      <div
        className="u-textCenter"
      >
        <h1
          className="u-spaceMT2 u-spaceMB8 u-md-spaceMB7 u-lg-spaceMB7 u-fontFamilyHeader u-fontSize5 u-md-fontSize3 u-lg-fontSize2"
        >
          Ladda ner appen
        </h1>
        <h2
          className="u-spaceMB10 u-md-spaceMB7 u-lg-spaceMB7 u-fontFamilyHeader u-fontSize8 u-md-fontSize7 u-lg-fontSize7"
        >
          Sök efter Hedvig i din Store för att komma igång.
        </h2>
      </div>
      <div
        className="u-spaceMB5"
      >
        <div
          className="u-textCenter"
        >
          <a
            className="Download__social-icon"
            href="https://itunes.apple.com/se/app/hedvig/id1303668531?mt=8"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Ladda ner på App Store"
              height={54}
              src="/assets/appstores/app-store-badge@2x.png"
            />
          </a>
          <a
            className="Download__social-icon"
            href="https://play.google.com/store/apps/details?id=com.hedvig.app"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Ladda ner på Google Play"
              height={54}
              src="/assets/appstores/google-play-badge@2x.png"
            />
          </a>
        </div>
      </div>
    </div>
  </article>
  <Footer />
</main>
`;
